orientation_readme.md

```diff
@@ -0,0 +1,82 @@
+# Orientation : Welcome to your first day!
+
+This document was made to help newcomers set up all the software that they will need to actively learn and participate in the JustAskDavidB Coaching Program.
+
+
+##  Downloading and Setting Up Software
+
+### [Jetbrain](https://account.jetbrains.com/login)
+We use Jetbrain products for web development.  
```

Comment:
After careful consideration, make all paragraphs one solid uninterrupting paragraph. The natural wrapping of text and the publication will adjust formatting in medium as appropriate.  And GitHub applies wrapping automatically on a per device basis. This should address the "leading space" issue.